autogen/agentchat/groupchat.py

```diff
@@ -162,6 +180,13 @@ def __post_init__(self):
             agents=self.agents,
         )
 
+        # Check select_speaker_message_template and select_speaker_prompt_template have values
+        if self.select_speaker_message_template is None or len(self.select_speaker_message_template) == 0:
+            raise ValueError("select_speaker_message_template cannot be empty or None.")
+
+        if self.select_speaker_prompt_template is None or len(self.select_speaker_prompt_template) == 0:
+            raise ValueError("select_speaker_prompt_template cannot be empty or None.")
+
```

Comment:
Contradicting with "Optional" in their typing?

Reply:
Yes, sorry, my initial code allowed them to be None and then later I changed them to default to the original templates.to include the original templates as defaults.

I'll change Optional[str] to just str.